maptivist/src/main/webapp/script.js

```diff
@@ -32,6 +32,64 @@ function initMap(){
     mapId: '837a93b1537b2a61'
   });
 
+  // EXAMPLE MARKER
```

Comment:
Do we need to add back this Example Marker still? It seems to cause problems since it doesn't exist in everyone's datastore.